index.html

```diff
@@ -62,42 +70,41 @@ <h1>I'm Chime Glad to see you!</h1>
 
     <div id="projects-section">
```

Comment:
- [ ] Change the `background color` of this section as in the original website, so the design will be easily seen.